apps/mobile/src/features/Signers/components/SignersList/SignersListItem.tsx

```diff
@@ -24,7 +24,15 @@ function SignersListItem({ item, index, signersGroup }: SignersListItemProps) {
   const colorScheme = useColorScheme()
   const contact = useAppSelector(selectContactByAddress(item.value))
   const local = useLocalSearchParams<{ safeAddress: string; chainId: string; import_safe: string }>()
-  const actions = useSignersActions()
+
+  // Check if the current item belongs to the 'My signers' section
+  const isMySigner = signersGroup.some(
+    (section) => section.title === 'My signers' && section.data.some((signer) => signer.value === item.value),
```

Comment:
Can we maybe add a key/id to the section instead? "My signers" were named something else at the begining and I can see it changing to something else. Plus, what if it is multilingual? Better to rely on a stable key than wonder why this condition is not working.